src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const reverseObj = {};
+
+  for (const key in robot) {
+    if (reverseObj.hasOwnProperty([robot[key]])) {
+      return null;
+    } else {
+      reverseObj[robot[key]] = key;
+    }
```

Comment:
else is redundant here
```suggestion
    reverseObj[robot[key]] = key;
```